src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -159,17 +165,17 @@ export const checkNewBoundaryEnabled = async (
   return next()
 }
 
-export const validateSubmission = async (
-  req: StorageSubmissionMiddlewareHandlerRequest,
+export const validateStorageSubmission = async (
```

Comment:
to add JSDocs here

Reply:
Added in [511413d](https://github.com/opengovsg/FormSG/pull/6696/commits/511413db4d9aadda5d229122c869ed5dbf995355), thanks!